src/hoplon/core.cljs

```diff
@@ -9,6 +9,7 @@
 (ns hoplon.core
   (:require
     [goog.Uri]
+    [goog.object    :refer [get set]]
```

Comment:
Doesn't this mask out `clojure.core/get`?
I think this could lead to confusion in the future.
As I see it's quite wide-spread to use the `[goog.object :as gobj]` alias, so I would just use `gobj/get` and `gobj/set` in these two locations.